src/Orleans.Hosting.ServiceFabric/FabricSiloInfo.cs

```diff
@@ -11,24 +12,6 @@
     public class FabricSiloInfo
```

Comment:
I see that this Json-serializable class is also use with IOptions. Should they really be the same thing? Haven't reviewed the whole thing to understand what's the JSON representation used for

Reply:
The JSON representation is used for Service Fabric's naming service.

The IOptions portion is mostly so that external developers could publish their own endpoints via the `OtherEndpoints` property.

Reply:
BTW, originally I thought of having two, but I figured it was unnecessary/over-engineering

Reply:
I'm not really convinced, but don't feel that strongly either, especially since we haven't full migrated off the legacy configuration and into Options. I think we can re-asses afterwards

Reply:
What's your preference for now? Change the PR or leave it?

Reply:
nah, leaving it for now it's OK. I was just waiting for the build to finish